alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -0,0 +1,25 @@
+CRD_SCHEMA_GEN_VERSION ?=v0.2.1
+CRD_SCHEMA_GEN_APIS ?=./vendor/github.com/openshift/api/operator/v1
```

Comment:
Don't store it into vendor it's easy to get whiped on unrelated changes. `pkg/api`?

Reply:
We're not storing anything in vendor, this is reading `openshift/api` from vendor. When it's run from our repos this is where the API is

Reply:
+1 

This vendor path is correct.

Reply:
I am not sure there should be such default if it is repo specific, set it in the repo, not in library

Reply:
@tnozicka you are right. This path here is wrong. It belong into the consumer repo. No default here.